rocAL/rocAL/include/readers/image/reader.h

```diff
@@ -159,6 +159,6 @@ class Reader {
     virtual std::string id() = 0;
     //! Returns the number of items remained in this resource
     virtual unsigned count_items() = 0;
-    
+    virtual std::string file_path() = 0;
```

Comment:
This has been introduced for audio source evaluation & audio_read_and_decode where we take the file path instead of the reader->id()